var/spack/repos/builtin/packages/costa/package.py

```diff
@@ -21,6 +21,7 @@ class Costa(CMakePackage):
     # note: The default archives produced with github do not have the archives
     #       of the submodules.
     version("master", branch="master", submodules=True)
+    version("2.1", sha256="c1e86452415083f7470b292d93ec60708b7c8dbafc2bac383636bb4b28135866")
```

Comment:
Looks like it's missing the release tarball https://github.com/eth-cscs/COSTA/releases

Version 2.0 has it